ambari/datadog_checks/ambari/ambari.py

```diff
@@ -77,6 +80,7 @@ def get_host_metrics(self, base_url, clusters, base_tags):
                         self._submit_gauge(metric_name, value, metric_tags)
```

Comment:
Here instead of having a tag `ambari_host`, you'd just have to do 
```python
self._submit_gauge(metric_name, value, metric_tags, hostname=host.get('Hosts').get('host_name'))
```
since you're now using external_tags

Reply:
Done